var/spack/repos/builtin/packages/py-sphinx-immaterial/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PySphinxImmaterial(PythonPackage):
+    """An adaptation of the popular mkdocs-material theme for the Sphinx documentation tool."""
+
+    homepage = "https://github.com/jbms/sphinx-immaterial"
+    pypi = "sphinx_immaterial/sphinx_immaterial-0.11.2.tar.gz"
+
+    version("0.11.2", sha256="a1c8387ca8b4da282949e474647d06f3b2f7d12fe54e9e662b962771012bf257")
+
+    depends_on("node-js@14:", type="build")
+    depends_on("npm", type="build")
```

Comment:
Where did you find these deps (and versions) listed? Might be worth adding a comment in case someone updates this package someday.

Reply:
It's here in the docs: https://jbms.github.io/sphinx-immaterial/#material-for-sphinx

I'll add a note.